webapp/src/DOMJudgeBundle/Resources/views/jury/partials/submission_list.html.twig

```diff
@@ -75,6 +79,20 @@
                             (&hellip;)
                         {%- endif -%}
                 </a></td>
+                {% if showExternalResult | default(false) %}
+                    {% if submission.externalJudgements.empty %}
+                        {% set externalJudgement = null %}
+                    {% else %}
+                        {% set externalJudgement = submission.externalJudgements.first %}
+                    {% endif %}
+                    <td><a href="{{ link }}">
+                            {% if externalJudgement is null or externalJudgement.result is empty %}
+                                {{- 'pending' | printResult(true, true) -}}
```

Comment:
I wonder if we should create a specific `printValidJuryResult` function instead of this not really nice to read `(true, true)` argument lists where you have to know what it is.